common/lib/xmodule/xmodule/modulestore/xml.py

```diff
@@ -411,8 +422,7 @@ def load_course(self, course_dir, course_ids, tracker):
         returns a CourseDescriptor for the course
         """
         log.debug('========> Starting course import from {0}'.format(course_dir))
```

Comment:
Does this handle "loading" a library as well? A lot of the variable names, and log messages, are specific to course. But then parent_xml can be "library.xml" or "course.xml".